dev/plugins/hu.elte.txtuml.export.cpp/src/hu/elte/txtuml/export/cpp/templates/structual/ConstructorTemplates.java

```diff
@@ -75,7 +77,7 @@ public static String initDef(String className, String body, List<Pair<String, St
 		return source.toString();
```

Comment:
It might be a very insignificant issue, but is `StringBuilder` needed in the methods above?